app/views/businessdetails/contacting_you_phone.scala.html

```diff
@@ -24,22 +24,22 @@
 @header = {
     @components.back_link()
     @errorSummary(f)
-    @heading("businessdetails.contactingyou.phone.title", "summary.businessdetails")
 }
 
+@supportingContent = { <p>@Messages("businessdetails.contactingyou.phone.lbl2")</p> }
+
 @main(
     title = Messages("businessdetails.contactingyou.phone.title") + " - " + Messages("summary.businessdetails"),
     heading = header
 ) {
 
-    <p>@Messages("businessdetails.contactingyou.phone.lbl2")</p>
-
     @form(f, controllers.businessdetails.routes.ContactingYouPhoneController.post(edit)) {
 
-        @fieldset(
-            f = f("contactingyou-fieldset"),
-            legend = "businessdetails.contactingyou.phone.title",
-            legendHidden = true,
+        @fieldsetWithErrorGroup(
```

Comment:
Apologies I missed this. The fieldsetWithErrorGroup is not required and we only need the input component. See here for a similar implementation https://github.com/hmrc/amls-frontend/blob/master/app/views/businessactivities/business_employees.scala.html

Reply:
Done